x-pack/plugin/esql/qa/server/src/main/java/org/elasticsearch/xpack/esql/qa/rest/EsqlSpecTestCase.java

```diff
@@ -271,7 +230,9 @@ protected final void doTest(String query) throws Throwable {
             builder.tables(tables());
         }
 
-        Map<?, ?> prevTooks = supportsTook() ? tooks() : null;
+        boolean checkTook = supportsTook() && rarely();
```

Comment:
About 20% of the time we check `took`. It's actually pretty expensive to build. Not terribly so, but it shaved some load off of the test.

Reply:
It's not our numbers that are expensive - it's the fact that we have to fetch the entire x-pack usage api.